week1/project/ecommerce-app/src/fake-data/all-categories.js

```diff
@@ -0,0 +1,24 @@
+
+// export default [
+//   "FAKE: electronics",
+//   "FAKE: jewelery",
+//   "FAKE: men's clothing",
+//   "FAKE: women's clothing",
+// ];
+
+const categories = [
+  "FAKE: electronics",
+  "FAKE: jewelery",
+  "FAKE: men's clothing",
+  "FAKE: women's clothing",
+];
+
+// const categories = [
+//   {id : 1, category: "FAKE: electronics",
+//    id : 2, category: "FAKE: jewelery",
+//    id : 3, category: "FAKE: men's clothing",
+//    id : 4, category: "FAKE: women's clothing",
+//   }
+// ]
```

Comment:
Commented code like this can always be removed and should not end up in production environments 🙂 

```suggestion
const categories = [
  "FAKE: electronics",
  "FAKE: jewelery",
  "FAKE: men's clothing",
  "FAKE: women's clothing",
];
```